emr-serverless-sample/README.md

```diff
@@ -0,0 +1,112 @@
+# Running EMR Serverless Jobs with Java
+
+We will run a Java Spark job on EMR Serverless using a simple Java "Hello World" example in this example.
+
+## Prerequisites
+
+* LocalStack
+* `aws` CLI & `awslocal` script
+* Docker
+* Java and Maven
+
+## Installation
+
+Before creating the EMR Serverless job, we need to create a JAR file containing the Java code. To do this, run the following command:
+
+```bash
+cd hello-world
```

Comment:
Looks like the generated JAR file is tiny (only ~2KB), so potentially we could add it to the repo, to make the sample self-contained (and then list this step as optional). 👍